autoload/killersheep.vim

```diff
@@ -98,6 +98,7 @@ endfunc
 func s:IntroFilter(id, key)
   if a:key == 's' || a:key == 'S'
     call s:Clear()
+	let s:time_start = reltime()
```

Comment:
broken indentation

Reply:
oh, yes, i used to use 'tab' .. :) 
i modified it, and some others places had such broken also, thx review.